app/mod_api/models.py

```diff
@@ -232,7 +234,7 @@ class MetaDataDB(db.Model):
     name = db.Column(db.TEXT, index=True)
     descriptor = db.Column(db.JSON)
     status = db.Column(db.TEXT, index=True, default='active')
-    private = db.Column(db.Boolean)
+    private = db.Column(db.BOOLEAN, index=True, default=False)
```

Comment:
You are changing the database without adding a migration. Those shoulde be fixed first.